examples/Web/Startup.cs

```diff
@@ -16,6 +16,9 @@ public override void ConfigureServices(IServiceCollection services)
                 ;
 
             services.AddHttpContextAccessor();
+            services.AddMvc()
+                .IgnoreControllersOfType<System.Web.Mvc.IController>()
```

Comment:
Would like to avoid **Unravel.AspNetCore.Mvc** taking a dependency on either MVC or WebApi, so this extension method can either receive an appropriate base type or we can implement an `IgnoreAspNetControllers()` that maybe checks implemented interface namespaces instead (looking for `System.Web.`)?